.docs/user-guide/automation.md

```diff
@@ -14,7 +14,32 @@ orchestration tools below.  Optionally, Docker is also listed in some examples.
 
 <br>
 ## Ansible
-ToDo
+With Ansible, installing the latest REX-Ray binaries can be accomplished by
+including the `codenrhoden.rexray` role from Ansible Galaxy.  The role accepts
+all the necessary variables to properly fill out your `config.yml` file.
+
+Install the role from Galaxy:
+
+```shell
+ansible-galaxy install codenrhoden.rexray
+```
+
+Example playbook for installing REX-Ray on GCE Docker hosts:
+
+```yaml
+- hosts: gce_docker_hosts
+  roles:
+  - { role: codenrhoden.rexray,
```

Comment:
Cool - yeah, it's valid.  And it can be a bit weird, because this YAML is consumed by Ansible, and they've made a few extensions to what they consider valid formats.  They take true YAML, but also some hybrid formats to make their playbooks look "cleaner" in their opinion.